packages/baseai/src/dev/utils/thread/get-few-shot-messages.ts

```diff
@@ -1,4 +1,4 @@
-import type { Pipe } from '@/dev/routes/beta/pipes/run';
+import type { Pipe } from '@/dev/routes/v1/pipes/run';
```

Comment:
Same. We should keep one Pipe type. The one in `packages/baseai/types/pipe.ts` is the correct one.